libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork.m

```diff
@@ -93,7 +93,7 @@ + (instancetype)currentNetwork {
 
 + (instancetype)networkForUserAccount:(SFUserAccount*)account {
     CSFNetwork *instance = nil;
-    if (!account.isTemporaryUser) {
+    if (account!=nil) {
```

Comment:
IMO since we have removed the concept of anonymous users, the result of this method should be nil in the absence of a user. We may have to add another method to get a global shared instance of CSFNetwork.